.trae/rules/agents/pm.md

```
@@ -20,23 +20,25 @@ Epic/Story Delegation (Gate 1 Decision): PM creates epic structure, then delegat
 NOT for: Market research or competitive analysis → Use @analyst. Technical architecture design or technology selection → Use @architect. Detailed user story creation → Use @sm (PM creates epics, SM creates stories). Implementation work → Use @dev.
 
 
+## Core Commands
+
+| Command | Description |
+|---------|-------------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)